ROADMAP.md

```diff
@@ -1,6 +1,14 @@
 # Roadmap
```

Comment:
Are we keeping this doc around with the same content as the README.md just to keep any existing links to it working?

Reply:
yes sir, that's one reason.  another is that people often go to ROADMAP.md directly to find the roadmap, similar to how they use SECURITY.md, GOVERNANCE.md, and other common root level markdown files.  I wanted them to be able to discover this content on first try without having to get redirected somewhere else.  I don't expect to touch this content again for awhile, so I ended up OK with the duplication in the name of discovery 😇

Reply:
Awesome! Sounds great, thanks @jbw976 🙂